pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -162,6 +162,9 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 	}
 
 	poolAndRadosNamespaceName := fmt.Sprintf("%s/%s", cephBlockPoolRadosNamespace.Spec.BlockPoolName, getRadosNamespaceName(cephBlockPoolRadosNamespace))
+	if getRadosNamespaceName(cephBlockPoolRadosNamespace) == "" {
+		poolAndRadosNamespaceName = cephBlockPoolRadosNamespace.Spec.BlockPoolName
+	}
```

Comment:
```suggestion
poolAndRadosNamespaceName := cephBlockPoolRadosNamespace.Spec.BlockPoolName
if rns:=getRadosNamespaceName(cephBlockPoolRadosNamespace) ;rns!= "" {
	poolAndRadosNamespaceName=fmt.Sprintf("%s/%s", cephBlockPoolRadosNamespace.Spec.BlockPoolName, rns)
}
```

we can avoid multiple `getRadosNamespaceName` call